<a href="https://colab.research.google.com/github/hizmarck/hacking-civico/blob/master/notebooks/02_python_101.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Primero:

**Densidad de población y relaciones con gatos públicos.**

Los datos fueron consultados desde https://www.inegi.org.mx/ de acuerdo al censo de población del 2015.

*Número de habitantes:*


* México = 119, 938 473 habitantes.
* Yucatán = 2, 097 175 habitantes.
* Mérida = 892 363 habitantes.


*Superficie:*



*   México = 1 960 189 km²
*   Yucatán = 43,379 km²
*   Mérida = 1528 km²

¿Cuál es la densidad poblacional?
















In [5]:
populationMx = 119938473;

def densidad_poblacional(poblacion, area_ciudad):
  return poblacion / area_ciudad

densityMx = densidad_poblacional(populationMx, 1960189)
densityYuc = densidad_poblacional(2097175, 43379)
densityMid = densidad_poblacional(892363, 1528)


# print results
print("Densidad en México = {} hab/km²".format(densityMx))
print("Densidad en Yucatán = {} hab/km²".format(densityYuc))
print("Densidad en Mérida = {} hab/km²".format(densityMid))


Densidad en México = 61.18719827526835 hab/km²
Densidad en Yucatán = 48.34539754258973 hab/km²
Densidad en Mérida = 584.0071989528795 hab/km²


¿Qué pasa al comparar varios estados o ciudades?

A medida que comparamos las ciudades sobre los estados, es posible observar que estas (las ciudades) concentran una gran cantidad de población.


# Segundo:

De acuerdo al sitio transparencia presupuestaria del gobierno de México, en el 2018 se tuvo *737,720.1* y en 2019 *746,742.8* millones de pesos respectivamente.


In [9]:
def gasto_percapita(gasto, poblacion):
  return gasto / poblacion

edu2018 = gasto_percapita(737720.1, populationMx)
edu2019 = gasto_percapita(746742.8, populationMx)

porcentaje = (edu2019 - edu2018) / edu2018 * 100

print("Gasto percapita en el 2018 = {}".format(edu2018))
print("Gasto percapita en el 2019 = {}".format(edu2019))
print("Porcentaje = {}".format(porcentaje))



Gasto percapita en el 2018 = 0.006150821179789407
Gasto percapita en el 2019 = 0.006226048917597943
Porcentaje = 1.2230519407021851


**¿En qué porcentaje aumentó o disminuyó?**

Aumentó un 1.22% con respecto al 2018.

**¿A qué crees que se deba?**

Pienso que el aumento fue tan minimo que podriamos decir que se mantuvo constante, y puedo decir que es el esquema de trabajo del actual gobierno, dado que se encontraba en la fase de austeridad republicana.


# Tercero:

**¿Para qué crees que nos pueda ser útil esta información?**

De un simple vistazo puede ser posible ver cuales son los objetivos del gobierno que se analice.

**¿Qué decisiones podría tomar el gobierno basado en ésta?**

Bueno, creo que el gobierno podría replantear estratégias que no le funcionaron en el pasado y corregirlas.





